.github/CODEOWNERS

```diff
@@ -30,18 +30,18 @@ go.sum @grafana/backend-platform
 /scripts/drone/ @grafana/grafana-release-eng
 
 # Cloud Datasources backend code
-/pkg/tsdb/cloudwatch @grafana/cloud-datasources @grafana/observability-squad
```

Comment:
did you intend to remove cloud datasources here as well?

Reply:
good catch, was not the plan 😄